src/server.c

```diff
@@ -7054,4 +7103,4 @@ int main(int argc, char **argv) {
     return 0;
 }
 
-/* The End */
+/* The End */
```

Comment:
```suggestion
/* The End */

```
I see a couple of other places you remove the newline at the end of the file, let's leave them there.